superset/databases/commands/test_connection.py

```diff
@@ -90,6 +91,14 @@ def run(self) -> None:  # pylint: disable=too-many-statements
             database.set_sqlalchemy_uri(uri)
             database.db_engine_spec.mutate_db_for_connection_test(database)
 
+            # Generate tunnel if present in the properties
+            ssh_tunnel = None
```

Comment:
nit, but I don't think you need this assignment here. ssh_tunnel will be None even if the next line evaluates to false. It will assign the value to the local scope.

Reply:
Fixed in the API PR pointing to this branch: https://github.com/apache/superset/pull/22199